Makefile

```diff
@@ -1,12 +1,28 @@
+# Variables for the project
+# =========================
+CONFERENCE ?= ep2024
```

Comment:
Is this `2024` correct?

Reply:
Yep, I haven't changed the questions yet so this is 2024. Will make a new branch for 2025 after these get merged.